change/@ni-nimble-components-e476eedb-989a-4246-811b-039ca8a8352e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Initial component bring up by integrating tiptap editor along with the footer formatting options",
```

Comment:
I unfortunately didn't get a chance to re-review today but I expect to be able to tomorrow